Makefile

```diff
@@ -5,6 +5,14 @@ endif
 
 # Install dependencies
 install:
+	@echo "Checking for PostgreSQL Installation..."
+	@if command -v psql > /dev/null 2>&1; then \
+		echo "PostgreSQL is already installed."; \
+	else \
+		echo "Installing PostgreSQL..."; \
+		(sudo apt-get update && sudo apt-get -y install postgresql postgresql-contrib) > /dev/null 2>&1 || exit 1; \
```

Comment:
we should not discard the installation logs because it becomes hard to understand if you system is stuck or if the installation is happening